BeeJet.Bot/Commands/Handlers/Steam/SyncSteamCommandHandler.cs

```diff
@@ -8,27 +10,44 @@ namespace BeeJet.Bot.Commands.Handlers.Steam
     public class SyncSteamCommandHandler : CommandSource
     {
         private SteamAPIService _steamAPI;
+        private readonly ISteamIdDiscordUserDb _steamUserDb;
+        private readonly BeeJetBotOptions _beeJetOptions;
 
-        public SyncSteamCommandHandler(Services.SteamAPIService steamAPI)
+        public SyncSteamCommandHandler(Services.SteamAPIService steamAPI, ISteamIdDiscordUserDb steamUserDb, BeeJetBotOptions options)
         {
             _steamAPI = steamAPI;
+            _steamUserDb = steamUserDb;
+            _beeJetOptions = options;
         }
 
         [BeeJetBotSlashCommand("sync-steam", "Sync steam library with channels", nameof(RegisterOptions))]
         public async Task SlashCommandExecuted()
         {
-            if (!ulong.TryParse((string)Context.SlashCommandInteraction.Data.Options.First().Value, out ulong steamId))
+            if (Context.SlashCommandInteraction.Data.Options.Count > 0 && !ulong.TryParse((string)Context.SlashCommandInteraction.Data.Options.First().Value, out ulong steamId))
             {
                 await Context.SlashCommandInteraction.RespondAsync("Not a valid steamid", ephemeral: true);
                 return;
             }
+            else
```

Comment:
Gezien je opzet met guard-clauses kan de else hier wel weg. Scheelt een niveautje :)

Reply:
Klopt het dat hij resolved is, maar de else er nog altijd is?

Reply:
Ja, maar mijn oplossing klopte nog steeds niet :-)

Probleem wat in deze versie er nog steeds was, als je steamid meegaf, ging hij gewoon zoeken naar je steam->discord link en die werd altijd leidend.